packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1478,8 +1488,8 @@ class Navigator extends StatefulWidget {
   ///
   /// The `T` type argument is the type of the return value of the popped route.
   ///
-  /// The type of `result`, if provided, must match the type argument of the
-  /// class of the popped route (`T`).
+  /// Returns true if a route was popped (including if [Route.didPop] returned
```

Comment:
The semantics of this are really odd. Remind me again why we need to keep the return value?

Reply:
Can we just tell people to use canPop if they need this information and remove the return value?